ExcelMapper/ExcelMapper.cs

```diff
@@ -358,6 +434,91 @@ IEnumerable Fetch(ISheet sheet, Type type)
             }
         }
 
+        IEnumerable<dynamic> FetchDynamic(ISheet sheet)
+        {
+            var firstRow = sheet.GetRow(HeaderRow ? HeaderRowNumber : MinRowNumber);
+
+            if (firstRow == null)
+                yield break;
+
+            var cells = Enumerable.Range(0, firstRow.LastCellNum).Select(i => firstRow.GetCell(i, MissingCellPolicy.CREATE_NULL_AS_BLANK));
+            var columns = cells
+                .Where(c => !HeaderRow || (c.CellType == CellType.String && !string.IsNullOrWhiteSpace(c.StringCellValue)))
+                .Select(c => new
+                {
+                    c.ColumnIndex,
+                    ColumnName = c.StringCellValue
+                })
+                .ToDictionary(c => c.ColumnIndex, c => c.ColumnName);
+
+            var i = MinRowNumber;
+            IRow row = null;
+
+            if (TrackObjects) Objects[sheet.SheetName] = new Dictionary<int, object>();
+
+            var objInstanceIdx = 0;
+
+            while (i <= MaxRowNumber && (row = sheet.GetRow(i)) != null)
+            {
+                // optionally skip header row and blank rows
+                if ((!HeaderRow || i != HeaderRowNumber) && (!SkipBlankRows || row.Cells.Any(c => !IsCellBlank(c))))
+                {
+                    dynamic o;
+                    IDictionary<string, object> dicoRow;
+                    o = dicoRow = new ExpandoObject();
+
+                    foreach (var col in columns)
+                    {
+                        var cell = row.GetCell(col.Key);
+
+                        if (cell != null && (!SkipBlankRows || !IsCellBlank(cell)))
+                        {
+                            var cellValue = GetCellValueDynamic(cell);
+                            dicoRow.Add(col.Key.ToString(), cellValue);// ByIndex
+                            if (HeaderRow)
+                                dicoRow.Add(col.Value, cellValue);// ByName
+                        }
+                    }
+
+                    if (TrackObjects) Objects[sheet.SheetName][i] = o;
+
+                    objInstanceIdx++;
+
+                    yield return o;
+                }
+
+                i++;
+            }
+        }
+
+        object GetCellValueDynamic(ICell cell, CellType? cellType = null)
```

Comment:
This duplication of code from `GetCellValue` should be avoided. I think this can be done by eliminating this method and passing a dummy `ColumnInfo` to `GetCellValue`.

Reply:
The logic is noticeably different because there is no ``ColumnInfo`. There is no `ColumnInfo` because there is no prior `Type` attached to it.